syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -267,10 +300,13 @@ def mul(self, shares: dict, other_shares, *args, **kwargs):
 
         return self._abstract_mul("mul", shares, other_shares, **kwargs)
 
-    def __mul__(self, *args, **kwargs):
+    def __mul__(self, other, **kwargs):
         """Multiplies two number for details see mul
         """
-        return self.mul(*args, **kwargs)
+        # FIXME
```

Comment:
Fixmee why?

Reply:
I think it's ok, even though we would prefer not to handle this particular case